app/main.py

```diff
@@ -22,6 +26,24 @@ def create_app() -> FastAPI:
 
     _app.add_exception_handler(ExceptionCustom, my_exception_handler)
 
+    
+    # Edit on the exceptiion handler of data type validation pydantic to send custom error messages
+    @_app.exception_handler(RequestValidationError)
```

Comment:
praise: nice work and great job for customizing the Exception!
suggestion: extract the handling logic into a function and move it to handling_exception file and import it from their